cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -393,31 +393,31 @@ share
   -- The table containing pools' on-chain reference to its off-chain metadata.
 
   PoolMetadataReference
-    poolId              Types.PoolId              sqltype=text
-    url                 Types.PoolUrl             sqltype=text
-    hash                Types.PoolMetadataHash    sqltype=text
+    poolId              Types.DbPoolId              sqltype=text
+    url                 Types.DbPoolUrl             sqltype=text
+    hash                Types.DbPoolMetadataHash    sqltype=text
     UniquePoolMetadataReference  poolId hash
 
   -- The table containing the metadata.
 
   PoolMetadata
-    poolId              Types.PoolId              sqltype=text
-    tickerName          Types.TickerName          sqltype=text
-    hash                Types.PoolMetadataHash    sqltype=text
-    metadata            Types.PoolMetadataRaw     sqltype=text
+    poolId              Types.DbPoolId              sqltype=text
+    tickerName          Types.DbTickerName          sqltype=text
+    hash                Types.DbPoolMetadataHash    sqltype=text
+    metadata            Types.DbPoolMetadataRaw     sqltype=text
```

Comment:
`tickerName` should be `Text`. `hash` and probably `metadata` should be `ByteString`.

`poolId` should only be named that if it is an index into the `pool_update` table (which it probably is).